packages/wallet-core/src/store/actions/updateNFTs.ts

```diff
@@ -1,20 +1,21 @@
 import Bluebird from 'bluebird';
 import { ActionContext, rootActionContext } from '..';
-import { Account, AccountId, Network, NFT, WalletId } from '../types';
+import { Account, Network, NFT, WalletId } from '../types';
 
 export const updateNFTs = async (
   context: ActionContext,
   {
     walletId,
     network,
-    accountIds,
   }: {
     walletId: WalletId;
     network: Network;
-    accountIds: AccountId[];
```

Comment:
Why is this changing? It should still be possible to update only a single account's NFTs